src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -931,5 +932,107 @@ public void WorkingDirectory_Set_GetReturnsExpected(string workingDirectory)
             var info = new ProcessStartInfo { WorkingDirectory = workingDirectory };
             Assert.Equal(workingDirectory ?? string.Empty, info.WorkingDirectory);
         }
+
+        [Fact(Skip = "Manual test")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void StartInfo_WebPage()
+        {
+            ProcessStartInfo info = new ProcessStartInfo
+            {
+                UseShellExecute = true,
+                FileName = @"http://www.microsoft.com"
+            };
+
+            Process.Start(info);
+        }
+
+        [Theory, MemberData(nameof(UseShellExecute))]
+        [OuterLoop("Launches notepad")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void StartInfo_NotepadWithContent(bool useShellExecute)
+        {
+            string tempFile = GetTestFilePath() + ".txt";
+            File.WriteAllText(tempFile, $"StartInfo_NotepadWithContent({useShellExecute})");
+
+            ProcessStartInfo info = new ProcessStartInfo
+            {
+                UseShellExecute = useShellExecute,
+                FileName = @"notepad.exe",
+                Arguments = tempFile,
+                WindowStyle = ProcessWindowStyle.Minimized
```

Comment:
Yeah, and unfortunately it is a little complicated. In this particular case it doesn't do anything as we're passing a file- notepad ignores the flag.  Really need to write a test app that we can fire up and validate that it is getting configured correctly.

In general we're lax on tests with Process- especially if the bugs I fixed are any indication. I'm going to try and follow up with more tests, but wanted to get the core scenario up for review.